### PR TITLE
Remove unneeded src imports to use web component

### PR DESCRIPTION
**Pull request by web-padawan**
There shouldn't be any need for `src` imports now when the web component is [doing the same](https://github.com/vaadin/vaadin-context-menu/blob/3bc6199f4f6a71adb531e6e804d7990a33ff9d5e/src/vaadin-contextmenu-items-mixin.html#L1-L2):

```html
<link rel="import" href="../../vaadin-item/src/vaadin-item.html">
<link rel="import" href="../../vaadin-list-box/src/vaadin-list-box.html">
```

These imports have been added while implementing the hierarchical support for Context Menu.

Also, `@NpmPackage` annotations should not be needed here, because `@vaadin/vaadin-context-menu` explicitly depends on both `@vaadin/vaadin-list-box` and `@vaadin/vaadin-item`.

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-context-menu-flow/90)

### CODE REVIEW

File: vaadin-context-menu-flow-integration-tests/webpack.generated.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
+/**
```
Comment (pekam) [inline vaadin-context-menu-flow-integration-tests/webpack.generated.js]: Looks like you committed this file by accident.

**Review by pekam — APPROVED** [review]

**Comment by pekam** [comment]
ping @ujoni @manolo

File: vaadin-context-menu-flow-integration-tests/webpack.generated.js (hunk 1 above)
Comment (web-padawan) [reply vaadin-context-menu-flow-integration-tests/webpack.generated.js]: Thanks, removed.

**Comment by ujoni** [comment]
This is most likely correct but I'd like  @manolo to verify this since he is _the_ expert.

**Review by manolo — APPROVED** [review]
:lgtm: 

> Reviewed 2 of 3 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/vaadin/vaadin-context-menu-flow/90#-:-LiDrVJWCtaMzC-AyEuV:bg7igkv)* status: all files reviewed, 1 unresolved discussion (waiting on @pekam)

**Review by web-padawan — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/vaadin/vaadin-context-menu-flow/90)* status: all files reviewed, 1 unresolved discussion (waiting on @pekam)

___
*[vaadin-context-menu-flow-integration-tests/webpack.generated.js, line 1 at r1](https://reviewable.io/reviews/vaadin/vaadin-context-menu-flow/90#-LiDgCs-WlHAGk6eokqq-r1-1:-LiDrqUv3ZUM0UCxRwe3:b-896fix) ([raw file](https://github.com/vaadin/vaadin-context-menu-flow/blob/a82fe46e1af869a3e1e8c29e32404f08c3d11e17/vaadin-context-menu-flow-integration-tests/webpack.generated.js#L1)):*
<details><summary><i>Previously, web-padawan (Serhii Kulykov) wrote…</i></summary><blockquote>

Thanks, removed.
</blockquote></details>

Done.

**Review by pekam — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/vaadin/vaadin-context-menu-flow/90)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**